src/dcqc/suites/suites.py

```diff
@@ -45,3 +45,7 @@ class BAMSuite(FileSuite):
 class FastqSuite(FileSuite):
     file_type = FileType.get_file_type("FASTQ")
     add_tests = (tests.PairedFastqParityTest,)
+
```

Comment:
Is there an EDAM format for TXT?

Reply:
yes, TXT was already in file.py along with its format ID